.gitignore

```diff
@@ -0,0 +1,35 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```

Comment:
May also consider whether some of the other hidden files need to be committed or not like .gitignore itself?
<img width="1001" alt="Screenshot 2022-04-25 at 08 01 57" src="https://user-images.githubusercontent.com/68547574/165038467-aaae4c6c-8b41-496b-80ea-03c422823f02.png">
<img width="1001" alt="Screenshot 2022-04-25 at 08 01 57" src="https://user-images.githubusercontent.com/68547574/165038467-aaae4c6c-8b41-496b-80ea-03c422823f02.png">